selfdrive/car/volkswagen/values.py

```diff
@@ -381,6 +385,7 @@ class VWCarInfo(CarInfo):
       b'\xf1\x875Q0907572D \xf1\x890304\xf1\x82\00101',
       b'\xf1\x875Q0907572E \xf1\x89X310\xf1\x82\00101',
       b'\xf1\x875Q0907572F \xf1\x890400\xf1\x82\00101',
+      b'\xf1\x875Q0907572F \xf1\x890400\xf1\x82\x0101',
```

Comment:
Don't add duplicates and insert in places to keep the lines sorted.

Reply:
I don't see any duplicates

Reply:
CI does: 
```Traceback (most recent call last):
  File "/tmp/openpilot/selfdrive/car/tests/test_fw_fingerprint.py", line 48, in test_no_duplicate_fw_versions
    self.assertTrue(passed, "Duplicate FW versions found")
AssertionError: False is not true : Duplicate FW versions found ```